gui/src/near/methods.ts

```diff
@@ -8,20 +8,37 @@ function hasContractMethodProperty(obj: {}): obj is {contractMethod: "change" |
   return 'contractMethod' in obj
 }
 
-function hasContractMethod(m: MethodName, equalTo?: "change" | "view") {
+function hasAllowProperty(obj: {}): obj is { allow: string[] } {
+  return 'allow' in obj
+}
+
+function hasContractMethod(m: MethodName, equalTo?: "change" | "view"): boolean {
   const def = topLevelSchema.definitions[m]
   const hasField = hasContractMethodProperty(def)
   if (!hasField) return false
   if (!equalTo) return true
   return def.contractMethod === equalTo
 }
 
-export const changeMethods = Object.keys(topLevelSchema.definitions).filter(
-  m => hasContractMethod(m as MethodName, "change")
+function allowsAdmin(m: MethodName): boolean {
+  const def = topLevelSchema.definitions[m]
+  const hasField = hasAllowProperty(def)
+  if (!hasField) return false
+  return def.allow.includes('::admins')
+}
+
+export const adminMethods = Object.keys(topLevelSchema.definitions).filter(m =>
+  allowsAdmin(m as MethodName)
+) as MethodName[]
```

Comment:
Plans for a follow-up PR:

- [ ] Since we want to make this form general-purpose for any contract, we should stop making assumptions about the permission system of a contract. Rather than have one group of `admins`, a contract could have, say, a group of `legislators` that can do some things and `executives` who can do others. Whatever. So the UI should just filter the list of Change Methods based on the current user's permissions.